### PR TITLE
[WIP] runner-api.adoc

### PR DESCRIPTION
**Pull request by thomasr8**
Initial draft of CircleCI Runner API Guide

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: jekyll/_cci2/runner-api.adoc [hunk 1]
```diff
@@ -0,0 +1,190 @@
+= CircleCI Runner API
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+//:circle-success: image:circle-success.png[]
+//:circle-warning: image:circle-warning.png[]
+//:circle-failure: image:circle-failure.png[]
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-api.adoc:9]: These images were for a specific server page so these lines can safely be removed :-)

File: jekyll/_cci2/runner-api.adoc [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+This document contains all the external facing endpoints for the Agent API.
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-api.adoc:11]: Should this be 'runner API'?

File: jekyll/_cci2/runner-api.adoc [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+| The token can be provided using the Basic scheme, where username should be set as the circle-token and the password should be left blank
+|===
+
+=== Browser Session Authentication
+
+Ring-session sent through a cookie on the request. This authentication allows users that are already logged into circleci.com to access certain endpoints seamlessly.
+
+=== Launch Token
+
+This token is the same one used in the launch-agent setup.
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-api.adoc:40]: Can we link to launch-agent setup for some context?

File: jekyll/_cci2/runner-api.adoc (hunk 1 above)
Comment (thomasr8) [reply jekyll/_cci2/runner-api.adoc]: Fixed

File: jekyll/_cci2/runner-api.adoc (hunk 2 above)
Comment (thomasr8) [reply jekyll/_cci2/runner-api.adoc]: Fixed

**Pull request merged**